app/controllers/state_file/intake_logins_controller.rb

```diff
@@ -75,7 +76,25 @@ def service_type
     end
 
     def redirect_to_data_review_if_intake_authenticated
-      redirect_to StateFile::Questions::DataReviewController.to_path_helper(us_state: params[:us_state]) if current_state_file_az_intake.present? || current_state_file_ny_intake.present?
+      intake = current_state_file_az_intake || current_state_file_ny_intake
+      if intake.present?
+        if intake.hashed_ssn.present?
+          redirect_to StateFile::Questions::DataReviewController.to_path_helper(us_state: params[:us_state])
+        else
+          redirect_to StateFile::Questions::TermsAndConditionsController.to_path_helper(us_state: params[:us_state])
+        end
+      end
+    end
+
+    def sign_in_and_redirect(controller)
+      intake = @records.first
```

Comment:
If this relation if filtering based on email or phone, we could do .take that will skip the order by in the SQL for a slight improvement.
```suggestion
      intake = @records.take
```

Reply:
what's the reason for picking the first one? i assume we just think they're all the same?